### PR TITLE
doc delete post api

### PR DESCRIPTION
**Pull request by at-haytran**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-114

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: docs/api-references/post.md [hunk 1]
```diff
@@ -91,3 +91,33 @@ Get list posts following book
...
+| --- | --- | --- |
+| id | Number | Id of post |
+#### Response success
+| Field | Type | Description |
+|---|---|---|
+| message | String | Status result |
+
+```json
+{
+    "status": "Successfully",
```
Comment (at-nale) [inline docs/api-references/post.md]: chuyển status thành message

File: docs/api-references/post.md [hunk 2]
```diff
@@ -137,4 +137,49 @@ Create new post
...
+|Accept|application\json|
+|Authorization|{token_type} {access_token}
+#### Parameters
+| Field | Type | Description |
+| --- | --- | --- |
+| id | Number | Id of post |
+#### Response success
+| Field | Type | Description |
+|---|---|---|
+| message | String | Status result |
```
Comment (at-quangtran) [inline docs/api-references/post.md]: Status result-> message result

File: docs/api-references/post.md [hunk 3]
```diff
@@ -137,4 +137,46 @@ Create new post
...
+|Accept|application\json|
+|Authorization|{token_type} {access_token}
+#### Parameters
+| Field | Type | Description |
+| --- | --- | --- |
+| id | Number | Id of post |
+#### Response success
+
+```json
+{
```
Comment (at-quangtran) [inline docs/api-references/post.md]: "status": 200,

Comment (at-haytran) [reply docs/api-references/post.md]: Bên cấu trúc api của tụi em không trả về status code anh.

**Pull request merged**